### PR TITLE
fix

### PR DESCRIPTION
**Pull request by Noanan**

### CODE REVIEW

File: pkg/infra/teams/team3/agent3.go [hunk 1]
```diff
@@ -71,8 +70,10 @@ func (a *AgentThree) UpdateInternalState(baseAgent agent.BaseAgent, _ *commons.I
 	if enable {
 		a.UpdatePersonality(baseAgent)
 	}
-	//fmt.Println(a.CalcBordaScore(baseAgent))
-	fmt.Println(a.SocialCapital(baseAgent))
+	// fmt.Println(a.CalcBordaScore(baseAgent))
+	a.CalcBordaScore(baseAgent)
+	// fmt.Println(a.SocialCapital(baseAgent))
+	a.SocialCapital(baseAgent)
```
Comment (MattSScott) [inline pkg/infra/teams/team3/agent3.go:76]: Why aren't the return values used?

**Review by MattSScott — COMMENTED** [review]
Minor thing with return vals

File: pkg/infra/teams/team3/agent3.go (hunk 1 above)
Comment (Noanan) [reply pkg/infra/teams/team3/agent3.go:76]: left them up to @Massoud26's implementation, this PR is just fixing bugs regarding the maps that weren't working

Comment (MattSScott) [reply pkg/infra/teams/team3/agent3.go:76]: sounds fair enough :)

**Review by MattSScott — APPROVED** [review]

File: 13-52-gameLog.csv [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+3,6,963,2030,20,20,47,0,0,1,5
+4,6,945,2030,20,20,46,0,0,1,5
+5,6,908,2060,20,20,47,0,0,2,4
+6,6,889,2070,20,20,49,0,1,1,4
+7,6,877,2080,20,20,49,0,0,1,5
+8,6,829,2060,20,20,49,0,0,0,6
+9,6,818,2090,20,20,51,0,0,1,5
+10,6,807,2100,20,20,52,0,2,1,3
+11,6,767,2110,20,20,53,0,0,0,6
+12,6,744,2100,20,20,52,0,0,0,6
```
Comment (antonypap) [inline 13-52-gameLog.csv]: maybe rm that file before merging it?

**Comment by antonypap** [comment]
Apart from my coment, LGTM

**Pull request merged**